### DISCUSSION TITLE
templates are not installing nuclei-templates  directory is blank.

### DISCUSSION THREAD

**Discussion by 0xmaruf** [projectdiscovery/nuclei#3636] [Q&A] [answered]
i reinstall go and nuclei latest version . still getting this below issue

```
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.2

                projectdiscovery.io

[INF] nuclei-templates are not installed, installing...
[INF] Your current nuclei-templates  are outdated. Latest is v9.4.3
[ERR] Could not find template '/home/kali/nuclei-templates': no templates found in path /home/kali/nuclei-templates
[INF] Current nuclei version: v2.9.2 (latest)
[INF] Current nuclei-templates version:  (outdated)
[INF] No results found. Better luck next time!
[FTL] Could not run nuclei: no valid templates were found
  ```

**Comment by ehsandeep** [comment]
@0xmaruf you can install templates to any other location with `nuclei -update-template-dir XXXX` option, you can replace with XXXX any location which is writable.

**Reply by 0xmaruf** [reply]
after using this command 
$ sudo nuclei -ud /home/kali -validate
i am getting this error now.

ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/cisco/ios/tests/integration/targets/ios_ospf_interfaces/tasks/cli.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!seq into templates.Alias
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/cisco/nxos/tests/integration/targets/nxos_user/tests/common/auth.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!seq into templates.Alias
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/dellemc/enterprise_sonic/tests/regression/roles/sonic_vlans/tasks/preparation_tests.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!seq into templates.Alias
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/junipernetworks/junos/tests/integration/targets/junos_snmp_server/tests/netconf/deleted.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!seq into templates.Alias
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/botocore/data/elastic-inference/2017-07-25/paginators-1.json: mandatory 'name' field is missing, mandatory 'author' field is missing, mandatory 'id' field is missing
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/fortinet/fortios/changelogs/.plugin-cache.yaml: yaml: unmarshal errors:
  line 1: field objects not found in type templates.Alias
  line 3: field plugins not found in type templates.Alias
  line 26: field version not found in type templates.Alias
[ERR] Error occurred loading template /home/kali/Desktop/cyrex/afrog-2.1.1/pocs/temp/afrog-pocs/vulnerability/Server-Side-JavaScript-Injection.yaml: yaml: unmarshal errors:
  line 11: field set not found in type templates.Alias
  line 13: field rules not found in type templates.Alias
  line 19: field expression not found in type templates.Alias
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/cisco/ios/tests/integration/targets/ios_acls/tasks/main.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!seq into templates.Alias
[ERR] Error occurred loading template /home/kali/.local/lib/python3.10/site-packages/ansible_collections/cisco/nxos/tests/integration/targets/nxos_bgp_address_family/tests/common/parsed.yaml: yaml: unmarshal errors:

**Reply by jdawg24** [reply]
depends on kali set up I think, try reinstall with standard user set up i.e. default location path at /home/user/nuclei-templates instead of ~/nuclei-templates maybe avoid xfce; install go and build nuclei as root

**Reply by 0xmaruf** [reply, marked as answer]
Yea it is fixed now,  i was having problem with zsh then i tried with bash and it worked.

**Reply by H1Xploit** [reply]
can you share the commands ?

**Comment by pb96692p** [comment]
i  have recently downloaded the nuclei and i did not find the  .local/nuclei-templates file hence i download it from git and ran 

$nuclei -l Desktop/mollielive.txt -t /home/atom69/.local/nuclei-templates/subdomain-takeover/ -v

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.2.4

                projectdiscovery.io

[VER] Started metrics server at localhost:9092
[WRN] Could not load template /home/atom69/.local/nuclei-templates/subdomain-takeover/detect-all-takeovers.yaml: yaml: line 210: mapping values are not allowed in this context
[INF] Current nuclei version: v3.2.4 (latest)
[INF] Current nuclei-templates version: v9.8.5 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] Targets loaded for current scan: 51
[INF] No results found. Better luck next time!
[FTL] Could not run nuclei: no templates provided for scan

can someone help me it is showing error in  detect-all-takeovers.yaml line 210
